javascript_nodejs/2.echobot-with-counter/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "echobot-with-counter",
+  "version": "1.0.0",
+  "description": "Bot Builder v4 echo bot sample",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha tests/*.test.js",
+    "start": "node app.js"
+  },
+  "author": "Microsoft",
+  "license": "MIT",
+  "dependencies": {
+    "botbuilder": "^4.0.0-preview1.2",
+    "botframework-config": "^4.0.0-preview1.2",
+    "dotenv": "^6.0.0"
+  }
```

Comment:
I'd like to add a devDependencies section to account for nodemon

    "devDependencies": {
        "nodemon": "^1.17.3",
}